components/brave_vpn/browser/brave_vpn_service.cc

```diff
@@ -187,12 +187,6 @@ bool BraveVpnService::IsConnected() const {
   return GetConnectionState() == ConnectionState::CONNECTED;
 }
 
-void BraveVpnService::RemoveVPNConnection() {
```

Comment:
need to delete this also from `brave_vpn_service.h`

Reply:
done